inode.go

```diff
@@ -0,0 +1,23 @@
+package mfs
+
+import (
+	ipld "github.com/ipfs/go-ipld-format"
+)
+
+// inode abstracts the common characteristics of the MFS `File`
+// and `Directory`. All of its attributes are initialized at
+// creation.
+type inode struct {
+	name       string
```

Comment:
Could you please put here the comments that were in the removed accessor functions? That way we could leverage them to document this structure.

Reply:
OK

Reply:
@schomatis it's ok, I had added the comments here.